tests/foreman/api/test_contentmanagement.py

```diff
@@ -1047,13 +1081,14 @@ def test_positive_sync_kickstart_repo(
 
         :BZ: 1992329
         """
+        distro = 'rhel8_aps'
```

Comment:
same concern - is this on purpose pinned to `rhel8` or did you want to parametrize the test to test all supported versions?

Reply:
Same as above, although this time it might be beneficial to parametrize and check kickstart-specific content for different repos. :thinking: